### PR TITLE
Add plugins.js and plugins.min.js and clean grunt/package.json things

### PR DESCRIPTION
**Pull request by mdamien**
Continuing ##210

* [x] removed necessity to include XLSX or dagre plugins with their third-party
* [x] removed gruntfiles for forceatlas and forcelink
   * it's working really well without it
* [x] fixed license and dependencies in `package.json` 
* [x] understand the old minification of forcealtas

Ideas:

* switch to a browserify workflow

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -164,7 +156,16 @@ module.exports = function(grunt) {
         },
         options: {
           sourceMap: true,
-          banner: '/* sigma.js - <%= pkg.description %> - Version: <%= pkg.version %> - Author: Alexis Jacomy, Sciences-Po Médialab - License: MIT */\n'
+          banner: '/* linkurious.js - <%= pkg.description %> - Version: <%= pkg.version %> - Author: Alexis Jacomy, Sciences-Po Médialab - License: GPLv3 */\n'
```
Comment (sheymann) [inline Gruntfile.js]: besoin d'éditer l'auteur

File: plugins/sigma.exporters.xlsx/sigma.exporters.xlsx.js [hunk 2]
```diff
@@ -220,6 +217,9 @@
    * @return {string}        The Spreadsheet string.
    */
   sigma.prototype.toXLSX = function(params) {
+      if (typeof XLSX === 'undefined')
```
Comment (sheymann) [inline plugins/sigma.exporters.xlsx/sigma.exporters.xlsx.js]: I'd prefer to throw error early if the file is included by the dependency is missing.

File: plugins/sigma.layout.dagre/sigma.layout.dagre.js [hunk 3]
```diff
@@ -89,6 +84,12 @@
    * ------------------
    */
   function dagreLayout() {
+    if (typeof dagre === 'undefined')
```
Comment (sheymann) [inline plugins/sigma.layout.dagre/sigma.layout.dagre.js]: I'd prefer to throw error early if the file is included by the dependency is missing.

File: plugins/sigma.exporters.xlsx/sigma.exporters.xlsx.js (hunk 2 above)
Comment (mdamien) [inline plugins/sigma.exporters.xlsx/sigma.exporters.xlsx.js]: It's so people can include the plugin without having to use it

File: Gruntfile.js (hunk 1 above)
Comment (mdamien) [inline Gruntfile.js]: tu peux commit ca ?

**Comment by mdamien** [comment]
Ok, the grunt task is here so the script don't have to optimize the forcelayout code each time it loads the script. But regex+string should be very fast....strange optim....

**Comment by mdamien** [comment]
Ready to be reviewed

File: plugins/sigma.exporters.xlsx/sigma.exporters.xlsx.js (hunk 2 above)
Comment (sheymann) [inline plugins/sigma.exporters.xlsx/sigma.exporters.xlsx.js]: ok

**Pull request merged**